src/apps/WalletConnect/hooks/useWalletConnect.tsx

```diff
@@ -53,22 +53,35 @@ const useWalletConnect = () => {
           throw error;
         }
 
-        if (payload.method === "eth_sendTransaction") {
-          const txInfo = payload.params[0];
-          safe.sendTransactions([
-            {
-              to: txInfo.to,
-              value: txInfo.value || "0x0",
-              data: txInfo.data || "0x",
-            },
-          ]);
-        } else {
-          wcConnector.rejectRequest({
-            id: payload.id,
-            error: {
-              message: "METHOD_NOT_SUPPORTED",
-            },
-          });
+        switch (payload.method) {
+          case "eth_sendTransaction": {
+            const txInfo = payload.params[0];
+            safe.sendTransactions([
+              {
+                to: txInfo.to,
+                value: txInfo.value || "0x0",
+                data: txInfo.data || "0x",
+              },
+            ]);
+            break;
+          }
+          case "gs_multi_send": {
+            const txs = payload.params as any[];
+            safe.sendTransactions(
+              // TODO: should we do some parameter validation here?
```

Comment:
The interface does its own validation and if doesn't pass, it displays an error message. I think it's ok to leave it like this

Reply:
does it make sense to validate that `data !== '0x' or value !== '0x0'` ?

Reply:
the question is if it makes sense to return there and not show the invalid tx modal

Reply:
See comment below